addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -922,6 +922,66 @@ QUnit.test('Attachments that have been unlinked from server should be visually u
     );
 });
 
+QUnit.test('The chatter should be cleared when creating a new record', async function (assert) {
+    assert.expect(3);
+
+    this.data['res.partner'].records.push( { display_name: "Partner1", id: 12 } );
+    this.data['mail.message'].records.push({
+        author_id: 12,
+        body: `test message`,
+        id: 1001,
+        model: 'res.partner',
+        res_id: 12,
+    }, {
+        author_id: 12,
+        body: `test message 2`,
+        id: 1002,
+        model: 'res.partner',
+        res_id: 12,
+    });
```

Comment:
messages are irrelevant to this test, only asserting the "Creating a new record" should be enough